src/cldfofflinebrowser/media.py

```diff
@@ -31,22 +31,22 @@ def read_media_files(cldf, filter=None):
     :return: A pair of `dict`s, the first mapping media ID to media row, the second mapping form ID\
     to `list` of related media IDs.
     """
+    form2media = collections.defaultdict(list)
+    media = {}
     try:
         form_ref = cldf['media.csv', 'formReference']
+        for r in cldf['media.csv']:
```

Comment:
You can't do it this way. The `try` `except` block from before was just checking whether a `formReference` exists - and even if not, the media table should be read - just no `form2media` was collected. So
- in case there's no media table: return `{}, {}`
- in case there's a media table but no formReference: return `{media files}, {}`

Reply:
right